LearningMissionLab/HelperSchedule.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LearningMissionLab
+{
+    public class HelperSchedule
```

Comment:
Please rename it to ScheduleEvent. Also, the fields are not correct. We discussed it already, the schedule event should have the following fields:
1. day of the week (Sunday, Monday, Tuesday, Wednesday, Thursday, Friday, Saturday)
2. start time hour: it could be anything from 0 to 24.
3. start time minutes: you can provide several predefined options here: 0 min, 15 min, 30 min, 45 min
4. duration (in minutes). You can again have predefined options here: 30 min, 45 min, 60 min, 90 min, 120 min, 150 min, 180 min.
OR if you prefer to use the end time instead of the duration, then you can specify the end time hour and minutes.